Sources/NostrSDK/Events/LongformContentEvent.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LongformContentEvent.swift
+//
+//
+//  Created by Bryan Montz on 11/2/23.
+//
+
+import Foundation
+
+public final class LongformContentEvent: NostrEvent {
```

Comment:
We should probably add documentation to the class and its attributes as well.

Reply:
Added. Thanks.